alicloud/resource_alicloud_slb_listener.go

```diff
@@ -82,6 +82,21 @@ func resourceAliyunSlbListener() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"acl_status": &schema.Schema{
+				Type:         schema.TypeString,
+				ValidateFunc: validateAllowedStringValue([]string{string(OnFlag), string(OffFlag)}),
+				Optional:     true,
+				Default:      OffFlag,
+			},
+			"acl_type": &schema.Schema{
+				Type:         schema.TypeString,
+				ValidateFunc: validateAllowedStringValue([]string{string(AclTypeBlack), string(AclTypeWhite)}),
+				Optional:     true,
+			},
+			"acl_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```

Comment:
I think `acl_type` and `acl_status` are valid when `acl_id` is set. So,  `acl_type` and `acl_status` need a DiffSuppressFunc